test/utils/granules.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+import test from 'ava';
+import { recoverAction } from '../../app/scripts/utils/table-config/granules';
+import { recoverGranule } from '../../app/scripts/actions';
+
+const granules = {
+  recovered: 'recovered'
+};
+
+test('recoverAction returns an empty list when disabled', function (t) {
```

Comment:
If you make the above recommended change, `recoverAction` itself will have no real logic built into it, so then I would do a more traditional component test where you mock the props/store to the granules overview component and test whether the recovery button is rendered

Reply:
Good idea. Working on it now.

Reply:
https://github.com/nasa/cumulus-dashboard/pull/568/commits/e2407e9d5aeb25d257b39f14f4ca266d34bc21ef